.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,227 @@
+name: GEOS CI 
+on: pull_request
+
+# Cancels in-progress workflows for a PR when updated
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  GEOSX_TPL_TAG: 220-932
+
+jobs:
+  # Matrix jobs will be cancelled if PR is a draft.
+  # PR status must be "Open" to run CI.
+  check_pull_request_is_not_a_draft:
+    # We use the most recent ubuntu distribution available in Github Actions to ensure maximum support of google cloud's sdk.
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check that the PR is not a draft (cancel rest of jobs otherwise)
+      run: |
+        echo "Is PR a draft?"
+        echo ${{ toJSON(github.event.pull_request.draft) }}
+        if [[ ${{ toJSON(github.event.pull_request.draft) }} == true ]]; then "false" ; else "true"; fi
```

Comment:
Could we use `if: github.event.pull_request.draft == false` instead?

Reply:
> Could we use if: github.event.pull_request.draft == false instead?

I gave that a try initially, but Github Actions takes a snapshot of the environment in each workflow.
So, let's say the draft job fails, so you change the PR from draft to Open.
When you re-run the failed job, it checks the saved draft state and fails again.

Reply:
Oh, this is interesting...